### PR TITLE
generate app.view.xml

### PR DESCRIPTION
**Pull request by lboehm**
Please take a look into my comment: https://github.com/SAP/open-ux-tools/issues/214#issuecomment-1066055414

The tests are still missing in this PR.

### CODE REVIEW

File: packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "sap.ui5": {
         "rootView": {
-            "viewName": "<%- app.id %>.view.<%- template.settings.viewName %>",
+            "viewName": "<%- app.id %>.view.App",
             "type": "XML",
             "async": true,
-            "id": "<%- template.settings.viewName %>"
+            "id": "App"
```
[the remark this answers is not in the document]
Comment (lboehm) [reply packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json:7]: In which config file should the parameter be stored? And do we really need to make this configurable? It's [best practice](https://github.com/ui5-community/generator-ui5-project/issues/21#issuecomment-997781079) to use `App.view.xml` and `App.controller.js`. When would we want to change this?

[the remark this answers is not in the document]
Comment (tobiasqueck) [reply packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json:7]: @lboehm when using the new Fiori flexible programming model, we require that you use (or extend) a root view provided by Fiori elements, that's why making the parameter configurable is useful.
I will push a suggestion on how to do it to your branch

[the remark this answers is not in the document]
Comment (tobiasqueck) [reply packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json:7]: Actually, I need to correct myself. The value should be static. There is no need to provide this as a customizable parameter. When using FPM, we anyhow need to do a bit more than just changing the app id, so they is no known reason to make this configurable.

**Review by tobiasqueck — APPROVED** [review]
* template enhanced as to guidance by UI5
* tests and test snapshots correctly updated
* valid changeset added

File: packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (lboehm) [reply packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json:7]: I think we can resolve this conversation, as the parameter can be static.

[the remark this answers is not in the document]
Comment (IainSAP) [reply packages/fiori-freestyle-writer/templates/basic/extend/webapp/manifest.json:7]: OK, so as I understand root view will always have a fixed name but "branch" views can still be configured with a custom view name. Thx.

**Review by IainSAP — APPROVED** [review]
Thx @lboehm .
- Verified that custom view name can still be provided for non-root view

**Pull request merged**